addons/binding/org.openhab.binding.enigma2/README.md

```diff
@@ -0,0 +1,52 @@
+# openHAB2 binding for Enigma2 STB
+
+This is the openHAB2 binding for an Enigma2 STB
+Here you can find a precompiled [Binding](https://github.com/tratho/org.openhab.binding.enigma2-dist)
```

Comment:
When it'll become an official bindnig in openHAB2, there's no need to link precompiled jar in README file :)